testing/suitesparse/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Maintainer: Jakub Jirutka <jakub@jirutka.cz>
+pkgname=suitesparse
+_pkgname=SuiteSparse
+pkgver=4.5.3
+pkgrel=0
+pkgdesc="A collection of sparse matrix libraries"
+url="http://faculty.cse.tamu.edu/davis/suitesparse.html"
+arch="x86 x86_64"
+license="GPL"
+depends_dev="openblas-dev"
+makedepends="$depends_dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="http://faculty.cse.tamu.edu/davis/$_pkgname/$_pkgname-$pkgver.tar.gz
+	0001-exclude-metis.patch
+	0002-remove-rpath.patch
+	"
+builddir="$srcdir/$_pkgname"
+
+prepare() {
+	cd "$builddir"
+
+	# OpenBLAS also provides LAPACK (?)
```

Comment:
I think Alpine's openblas is built with lapack.
And as I understand openblas is faster and more optimized, so better use it than standard lapack, right?